change image file name handling

anagha-todalbagi opened the pull request:
This commit takes care of reading files only with .jpg extensions from the sciencenterimages folder from the user's desktop for populating the arrays with the image file name and the scientist name for further use

anagha-todalbagi committed:
change image file name handling

chochim commented on ProjectX/ProjectX/MainWindow.xaml.cs:
```diff
@@ -30,8 +30,10 @@ namespace ProjectX
-        private string[] NAMES = { "alzheimer", "bailey", "bethe", "blackwell", "bolton", "buck", "cerrache" };
+        //private string[] IMAGES = { "projectx_4.jpg", "projectx_7.jpg", "projectx_12.jpg", "projectx_13.jpg", "projectx_15.jpg", "projectx_17.jpg", "projectx_22.jpg" };//, "projectx_32.jpg", "projectx_39.jpg" };//, "projectx_43.jpg", "projectx_44.jpg", "projectx_45.jpg", "projectx_55.jpg", "projectx_61.jpg" };    // images
+        //private string[] NAMES = { "alzheimer", "bailey", "bethe", "blackwell", "bolton", "buck", "cerrache" };
+        public string[] IMAGES = new string[98];
```
The number 98 is still hard-coded.
Just declare 
`public string[] IMAGES;`

you can dynamically allocate laterr

chochim commented on ProjectX/ProjectX/MainWindow.xaml.cs:
```diff
@@ -450,18 +452,20 @@ void _timer_Tick(object sender, object e)
         private void addImages()
         {
+            string desktopPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop);
+            string[] fileArray = Directory.GetFiles((@desktopPath + "\\sciencenterimages\\"), "*.jpg");
```
Here you can initialize the IMAGES and NAMES Array as
`IMAGES = new string[fileArray.Length];`

chochim reviewed (commented):
Please make the changes mentioned

anagha-todalbagi committed:
image array size decided based on number of images

This update introduces a static variable, imagesCount which finds the number of .jpg file available in the sciencenterimages file on the user's desktop and defines the size of the IMAGES and NAMES arrays to store the names of the image files and the last names of the scientists, respectively
